Makefile

```diff
@@ -169,15 +180,19 @@ setup-kind:
 # and run a test that does nothing but installs everything for
 # the elemental operator (nginx, rancher, operator, etc..) as part of the BeforeSuite
 # So you end up with a clean cluster in which nothing has run
-setup-full-cluster: build-docker-operator build-docker-seedimage-builder chart setup-kind
+setup-full-cluster: build-docker-operator chart setup-kind
 	@export EXTERNAL_IP=$$(kubectl get nodes -o jsonpath='{.items[].status.addresses[?(@.type == "InternalIP")].address}') && \
 	export BRIDGE_IP="172.18.0.1" && \
 	export CHART=$(CHART) && \
 	export CONFIG_PATH=$(E2E_CONF_FILE) && \
 	kind load docker-image --name $(CLUSTER_NAME) ${REGISTRY_HEADER}${REPO}:${CHART_VERSION} && \
-	kind load docker-image --name $(CLUSTER_NAME) ${REGISTRY_HEADER}${REPO_SEEDIMAGE}:${TAG_SEEDIMAGE} && \
+	kind load docker-image registry.opensuse.org/isv/rancher/elemental/dev/containers/rancher/seedimage-builder:1.5.0 --name operator-e2e && \
 	cd $(ROOT_DIR)/tests && $(GINKGO) -r -v --label-filter="do-nothing" ./e2e
 
+#  FIXME: Locally built seedimage builder image uses too old version of xorriso
```

Comment:
Is there an issue to try and fix the seedimage-builder?

Reply:
Added it. Thank you.